codersdk/users.go

```diff
@@ -104,15 +110,26 @@ type LoginWithPasswordResponse struct {
 	SessionToken string `json:"session_token" validate:"required"`
 }
 
+type OauthConversionResponse struct {
```

Comment:
How would `LoginType` not be `OIDC` but this name `OauthConversionResponse` still make sense?

I feel like we could optimize this abstraction with naming a bit.

Reply:
It can be `github` or `oidc`. Our BE treats them differently atm, so I made sure to specify here too.

This is primarily for the UI to know what to display.